libraries/c_sdk/standard/mqtt/src/iot_mqtt_network.c

```diff
@@ -64,9 +69,9 @@ static bool _incomingPacketValid( uint8_t packetType );
  *
  * @return #IOT_MQTT_SUCCESS, #IOT_MQTT_NO_MEMORY or #IOT_MQTT_BAD_RESPONSE.
  */
-static IotMqttError_t _getIncomingPacket( void * pNetworkConnection,
-                                          const _mqttConnection_t * pMqttConnection,
-                                          _mqttPacket_t * pIncomingPacket );
+static IotMqttError_t _getIncomingPacket(void* pNetworkConnection,
+    const _mqttConnection_t* pMqttConnection,
+    _mqttPacket_t* pIncomingPacket);
```

Comment:
is this the correct uncrustify used? It seems like the earlier formatting was correct.